src/code/sys_matrix.c

```diff
@@ -88,9 +88,9 @@ void SysMatrix_StatePop(void) {
 
 #pragma GLOBAL_ASM("asm/non_matchings/code/sys_matrix/SysMatrix_InsertRotationAroundUnitVector_s.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/sys_matrix/func_80182C90.s")
+#pragma GLOBAL_ASM("asm/non_matchings/code/sys_matrix/SysUcode_GetUCodeBoot.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/sys_matrix/func_80182CA0.s")
+#pragma GLOBAL_ASM("asm/non_matchings/code/sys_matrix/SysUcode_GetUCodeBootSize.s")
 
 #pragma GLOBAL_ASM("asm/non_matchings/code/sys_matrix/func_80182CBC.s")
```

Comment:
This doesn't look like part of the same file now. 🤔

Reply:
It looks like this should be `sys_ucode but we already have that file... A file split probably went wrong

Reply:
Looks like this was addressed in #274.